hazelcast/src/main/java/com/hazelcast/nio/serialization/SerializationServiceImpl.java

```diff
@@ -115,6 +119,7 @@ public Object getPartitionKey(Object key) {
         this.managedContext = managedContext;
         this.globalPartitioningStrategy = partitionStrategy;
         this.outputBufferSize = initialOutputBufferSize;
+        this.overrideCustomSerialization = parseBoolean(System.getProperty(SERIALIZATION_CUSTOM_OVERRIDE, "false"));
```

Comment:
just a trivial readability thing: you can use `Boolean.getBoolean()` method for this.